### ISSUE TITLE
"Slide" in the main view controller

### ISSUE THREAD

**Issue by sayakb** [dekatotoro/SlideMenuControllerSwift#69]
First of all, thank you for the awesome library. Saved me tons of effort.

I'm wondering if there's an easy way of adding a "slide from right" or "slide from left" animation to the main view controller. Or is my only option to add the animation to the slideMenuController's code directly?

**Comment by dhiaeldeen** [comment]
sayakb, i second you voice, i wish someone adds this feature.

**Comment by abhinavnair** [comment]
Yes currently, there is no animation in changeMainViewController. If we are switching from the Left Menu, then because the left menu slides back animated, it looks fine. But if we want to directly switch from any MainController, it is just an instantaneous view change.

**Comment by rcobelli** [comment]
I added a pull request for this #177
